src/React.AspNet/HtmlHelperExtensions.cs

```diff
@@ -63,7 +63,10 @@ private static IReactEnvironment Environment
 			bool clientOnly = false,
 			bool serverOnly = false,
 			string containerClass = null,
-			Action<Exception, string, string> exceptionHandler = null
+			Action<Exception, string, string> exceptionHandler = null,
+			Action<Func<string, string>> preRender = null,
+			Func<string, string> transformRender = null,
+			Action<Func<string, string>> postRender = null
```

Comment:
This function signature is getting a bit long... But I guess it's OK since they're all optional.

Reply:
I'll see about moving these into their own type, since they are all optional render settings.

Reply:
I ended up moving most of these into a separate interface